docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -191,8 +191,8 @@ Adaptive Batch Scheduler will only decide parallelism for operators whose parall
 
 - **Batch jobs only**: Adaptive Batch Scheduler only supports batch jobs. Exception will be thrown if a streaming job is submitted.
 - **ALL-EXCHANGES-BLOCKING jobs only**: At the moment, Adaptive Batch Scheduler only supports jobs whose [shuffle mode]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) is `ALL-EXCHANGES-BLOCKING`.
-- **The decided parallelism can only be a power of 2**: In order to make the subpartitoins evenly consumed by downstream tasks, user should configure the [`jobmanager.adaptive-batch-scheduler.max-parallelism`]({{< ref "docs/deployment/config" >}}#jobmanager-adaptive-batch-scheduler-max-parallelism) to be a power of 2 (2^N), and the decided parallelism will also be a power of 2 (2^M and M < N).
-- **No support for serveral file APIs**: No support for `StreamExecutionEnvironment#readFile` `StreamExecutionEnvironment#readTextFile` `StreamExecutionEnvironment#createInput(FileInputFormat, ...)` and all data sources using these APIs. When using these APIs, there will be a separate monitoring task (called a `Custom File Source`) as a predecessor to the actual data sources, which Adaptive Batch Scheduler cannot handle.
+- **The decided parallelism will be a power of 2**: In order to ensure downstream tasks to consume the same count of subpartitions, the configuration option [`jobmanager.adaptive-batch-scheduler.max-parallelism`]({{< ref "docs/deployment/config" >}}#jobmanager-adaptive-batch-scheduler-max-parallelism) should be set to be a power of 2 (2^N), and the decided parallelism will also be a power of 2 (2^M and M <= N).
+- **FileInputFormat sources are not supported**: FileInputFormat sources include `StreamExecutionEnvironment#readFile(...)` `StreamExecutionEnvironment#readTextFile(...)` and `StreamExecutionEnvironment#createInput(FileInputFormat, ...)` are not supported. Users should use the new sources([FileSystem DataStream Connector]({{< ref "docs/connectors/datastream/filesystem.md" >}}) or [FileSystem SQL Connector]({{< ref "docs/connectors/table/filesystem.md" >}})) to read files when using the Adaptive Batch Scheduler.
```

Comment:
I would state it like "FileInputFormat sources are not supported , including ..."